### DISCUSSION TITLE
How should I get a heatmap output from a converted mmpose model?

### DISCUSSION THREAD

**Discussion by starashima** [open-mmlab/mmdeploy#1306] [Q&A]
Hi. I could successfully get a keypoint output from a converted mmpose model (i.e., end2end.engine generated from [the default HRNet-W48 config](https://github.com/open-mmlab/mmpose/blob/master/configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/coco/hrnet_w48_coco_256x192.py) ), but I also want to get a corresponding heatmap output as I can get using [mmpose api](https://mmpose.readthedocs.io/en/latest/api.html#mmpose.apis.inference_top_down_pose_model).
How should I do this?

Thank you very much for your help.

**Comment by lvhan028** [comment]
@RunningLeon

**Comment by RunningLeon** [comment]
@starashima Hi, actually the trt engine only outputs heatmap. the decoding part is added in the End2EndModel. You can directly get the heatmap in here.
https://github.com/open-mmlab/mmdeploy/blob/4d4c10a2dc017e25343a5703b1037d3d45fa3bff/mmdeploy/codebase/mmpose/deploy/pose_detection_model.py#L95
